### PR TITLE
DOCSP-40790: Write operations landing page

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-40790
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/cpp-driver/DOCSP-40790-write-landing/write/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write.txt [hunk 1]
```diff
@@ -4,10 +4,143 @@
...
+----------
+
+.. literalinclude:: /includes/usage-examples/write-code-examples.cpp
+   :start-after: start-bulk-write
+   :end-before: end-bulk-write
+   :language: cpp
+   :dedent:
+
+To learn more about the ``create_bulk_write()`` method, see the
+:ref:`Bulk Write <cpp-bulk-write>` guide.
```
Comment (norareidy) [inline source/write.txt:146]: Note to reviewer: link will be broken until #37 is merged

File: source/write.txt [hunk 2]
```diff
@@ -4,10 +4,143 @@
...
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the {+driver-short+} to write data to MongoDB.
```
Comment (rustagir) [inline source/write.txt:18]: S: the source constant might not get parsed here
```suggestion
   :description: Learn how to use the C++ Driver to write data to MongoDB.
```

File: source/write.txt [hunk 3]
```diff
@@ -4,10 +4,143 @@
...
+
+.. literalinclude:: /includes/usage-examples/sample-app.cpp
+   :language: cpp
+   :dedent:
+   :linenos:
+   :emphasize-lines: 23-25
+
+Insert One
+----------
+
```
Comment (rustagir) [inline source/write.txt:62]: S: I think it's good practice to always introduce code examples. See my PR for kotlin sync for intros you can copy over https://github.com/mongodb/docs-kotlin-sync/pull/5
Applies to all sections

File: source/includes/usage-examples/write-code-examples.cpp [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        auto update_doc = make_document(kvp("$set", make_document(kvp("<field name>", <value>))));
+
+        auto result = collection.update_one(query_filter.view(), update_doc.view());
+        // end-update-one
+    }
+
+    {
+        // Updates all documents that match the specified criteria
+        // start-update-multiple
+        auto query_filter = make_document(kvp("<field to match>", <value to match>));
```
Comment (rustagir) [inline source/includes/usage-examples/write-code-examples.cpp:53]: S: for the sake of the syntax highlighting,  make the second param a string (applies to all)
```suggestion
        auto query_filter = make_document(kvp("<field to match>", "<value to match>"));
```

**Review by rustagir — CHANGES_REQUESTED** [review]
Main feedback is to add intro sentences!

**Review by rustagir — APPROVED** [review]
looks great! thanks for making changes to the read page as well.

**Review by kevinAlbs — APPROVED** [review]

**Pull request merged**